codegen/CMakeLists.txt

```diff
@@ -1,22 +1,29 @@
+cmake_minimum_required(VERSION 3.16)
+project(composable_kernel_host LANGUAGES CXX HIP)
+find_package(ROCM)
+include(ROCMInstallTargets)
+include(ROCMTest)
```

Comment:
This will break the main builds.

Reply:
I'm cognizant of that, this is just temporary to enable building codegen separately to ease testing if changes break anything. This will be removed before the PR is opened.